sickchill/oldbeard/providers/kat.py

```
@@ -72,20 +72,20 @@ def search(self, search_strings):
                     if self.url in self.disabled_mirrors:
-                        logger.info("Could not find a better mirror to try.")
-                        logger.info("The search did not return data, if the results are on the site maybe try a custom url, or a different one")
+                        logger.info(_("Could not find a better mirror to try."))
+                        logger.info(_("The search did not return data, if the results are on the site maybe try a custom url, or a different one"))
                         return results
 
                     # This will recurse a few times until all of the mirrors are exhausted if none of them work.
                     return self.search(search_strings)
 
                 with BS4Parser(data) as html:
                     labels = [cell.get_text() for cell in html.find(class_="firstr")("th")]
```

Ruff (0.8.2)

Local variable `labels` is assigned to but never used

Remove assignment to unused variable `labels`

(F841)